dvc/dependency/param.py

```diff
@@ -40,7 +40,7 @@ def __init__(self, stage, path, params):
             info=info,
         )
 
-    def _dyn_load(self, values=None):
+    def inject_values(self, values=None):
```

Comment:
Renamed this function

Reply:
@skshetry Still looks odd. Maybe `reload/parse/etc`? :) Inject sounds strange.

Reply:
`inject` is a correct term here. But, it does look strange. `reload/parse` is not what it's doing. Could go with `load_values` or `fill_values`.

Reply:
`fill_values` sounds better to me than `inject`.

Reply:
Done.